### ISSUE TITLE
for Enginseer(SphereOfInfluence food disappear bug)

### ISSUE THREAD

**Issue by bigcat88** [LoneGazebo/Community-Patch-DLL#3689]
[food dissapear.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1493162/food.dissapear.zip)

It is save from game(as I remember I don't use any mods except poortiles tweak, so u can load just with WorldCongress mod only), I placed destroyer near CS, just conquer it and install sphere of influence.
Each turn food will disappear from City tiles(4 turns is enough for all my cities in that save except capital)

Or if u want, delete destroyer and Oda Nobunaga will install his Sphere Of Influence(in one or two turns) and u will see how Japan lost food.

Don't look at my stables in that save, I added negative tourism to them, it is a watching fun game, I just sit on island and look on world waiting for this bug to appear.

@TechpriestEnginseer

**Comment by TechpriestEnginseer** [comment]
I still can't replicate. I can't explain why exactly.

**Comment by bigcat88** [comment]
I post a savegame, can u load it? If yes - do you see about what I saying in that savegame? If yes again -youu do you need more different saves?
I can try to catch this on tiny map 1vs1 in a 2-3 days...

**Comment by TechpriestEnginseer** [comment]
If I can't replicate it on a reproducible basis(A common factor in multiple scenarios that's causing the bug), I can't exactly fix it.

**Comment by bigcat88** [comment]
I added a string 
`Players[citystateliberate]:ChangeMinorCivFriendshipWithMajor(pPlayer:GetID(), OgreMagi(90, 100))`
after `if iChoice == iSphereofInfluence then`
and now starting a new game, will see what happens. Usually I see this bug every 3 games, so if the problem was in negative influence - in six days I will know.

**Comment by TechpriestEnginseer** [comment]
Still see no difference, so..

**Comment by bigcat88** [comment]
Was busy on work previous week, so will start a game only today.
What does this code do? 
`		if isUsingUCS then
			local minorPlayer = Players[citystateliberate]
			local majorPlayer = Players[iPlayer]
			if majorPlayer:GetEventChoiceCooldown(GameInfoTypes["PLAYER_EVENT_CHOICE_" .. GameInfo.MinorCivilizations[minorPlayer:GetMinorCivType()].Type]) == 0 then
				majorPlayer:DoEventChoice(GameInfoTypes["PLAYER_EVENT_CHOICE_" .. GameInfo.MinorCivilizations[minorPlayer:GetMinorCivType()].Type])	
			end
		end`
Will it work when "bad events" is disabled? Cause I play sometimes with "disabled bad events"...

**Comment by bigcat88** [comment]
if u need I have save from last patch, with the same. and you are right, adding ChangeMinorCivFriendshipWithMajor don't help me (

**Comment by TechpriestEnginseer** [comment]
I don't know and I still can't replicate it?

**Comment by bigcat88** [comment]
Ok, I replicated it in tiny map(with clear Vox hotfix install). Here is the save.
What was done:
1. Opened printing press.
2. Found city for myself.
3. Gave AI a settler, she found city too.
4. Wait till 1 world congress session passed.
5. Save before I declare war and liberating city.
[Augustus Caesar_0022 BC-3450.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1544810/Augustus.Caesar_0022.BC-3450.zip)
